modules/gdscript/language_server/godot_lsp.h

```diff
@@ -236,7 +236,7 @@ struct ReferenceContext {
 	/**
 	 * Include the declaration of the current symbol.
 	 */
-	bool includeDeclaration;
+	bool includeDeclaration = false;
```

Comment:
Fixes this:
![image](https://github.com/godotengine/godot/assets/4701338/76894bd1-f248-420d-90c6-e48b60e9fec2)

But this is actually never set anywhere, so the code where it was being used is bogus.

It was added in #80973, CC @ryanabx.